setup.py

```diff
@@ -4,7 +4,9 @@
     from setuptools import find_namespace_packages
 
     main_entrypoint = "pulp=pulpcore.cli.common:main"
-    plugin_packages = find_namespace_packages(include=["pulpcore.cli.*"])
+    plugin_packages = find_namespace_packages(
+        include=["pulpcore.cli.*"], exclude=["pulpcore.cli.*.*"]
+    )
```

Comment:
Does this method think the translation files are packages?

Reply:
This was listing all submodules too.